Makefile

```diff
@@ -17,9 +17,9 @@ node_modules/@financial-times/rel-engage/index.mk:
 
 # End generated by rel-engage
 # ---------------------------
-LOCAL_BOLT_URL=bolt://localhost:7687
+LOCAL_BOLT_URL=neo4j://localhost:7687
 BIZ_OPS_BOLT_URL=${NEO4J_BOLT_URL}
-NEO4J_VERSION=3.5.0
+NEO4J_VERSION=4.1.5
```

Comment:
There is now 4.1.6

Reply:
Upgraded to 4.1.6 (all three places) - thanks